java-security/src/main/java/com/sap/cloud/security/token/validation/validators/XsuaaJwtSignatureValidator.java

```diff
@@ -0,0 +1,60 @@
+package com.sap.cloud.security.token.validation.validators;
+
+import com.sap.cloud.security.config.OAuth2ServiceConfiguration;
+import com.sap.cloud.security.token.Token;
+import com.sap.cloud.security.xsuaa.client.OAuth2ServiceException;
+
+import java.net.URI;
+import java.security.NoSuchAlgorithmException;
+import java.security.PublicKey;
+import java.security.spec.InvalidKeySpecException;
+
+import static com.sap.cloud.security.token.validation.validators.JsonWebKeyConstants.*;
+
+class XsuaaJwtSignatureValidator extends JwtSignatureValidator {
+    public static final String FALLBACK_KEY = "verificationkey";
```

Comment:
we have a constant already for the verifcation key you dont have to redefine it again